content/en/blog/releases/release-0.6.0.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Release 0.6.0"
+linkTitle: "Release 0.6.0"
+date: 2022-05-23
+description: >
+  What is new in Version 0.6.0?
+---
+
+**We are pleased to announce the release of Merbridge 0.6.0!**
+
+In this release, we have introduced CNI mode for the first time to support the full amount of Istio traffic forwarding related capabilities. For details, you can refer to [Merbridge CNI Mode](/blog/2022/05/18/cni-mode/)
+
+Please refer to the details of the revision: [Merbridge 0.6.0](https://github.com/merbridge/merbridge/releases/tag/0.6.0)
```

Comment:
For release notes, see [Merbridge 0.6.0](https://github.com/merbridge/merbridge/releases/tag/0.6.0).